src/extension.ts

```diff
@@ -130,3 +154,38 @@ function getSnippets(snippetsJson: any): vscode.CompletionList {
 // This method is called when your extension is deactivated
 export function deactivate() { }
 
+function watchL10nYaml(): vscode.FileSystemWatcher | undefined {
+	const folder = vscode.workspace.workspaceFolders?.[0];
```

Comment:
Using only the first workspace folder here might lead to unpredictable behaviour in multi-root workspaces. For example if you open ProjectA + ProjectB you could get different behaviour to opening ProjectB on its own.

~You could use `workspace.getWorkspaceFolder(Uri file)` to get the correct workspace folder for any given file in the workspace, although you can't just track a single set of options.

It might also not be clear that this file goes in the folder that you open in VS Code (so if you open a folder that contains multiple projects, the file would be at the root and not within each project).

Perhaps it would be clearer if this was a real VS Code extension setting (at the workspace level). That way it shows in the VS Code settings UI and it's clear that you can set it in User Settings or at Workspace level (where it would be in `.vscode/settings.json` or the `.code-workspace` file)?